frontend/src/app/main-dashboard/main-dashboard.component.html

```diff
@@ -12,29 +12,33 @@
 See the License for the specific language governing permissions and
 limitations under the License. -->
 
-<!-- Render plot. -->
-<app-plot id="main-plot"></app-plot>
-<div class="grid-container">
-  <mat-grid-list cols="10" rowHeight="100px">
-    <mat-grid-tile *ngFor="let card of cards | async" [colspan]="card.cols" [rowspan]="card.rows">
-      <mat-card class="dashboard-card">
-        <mat-card-header>
-          <mat-card-title>
-            {{card.title}}
-            <button mat-icon-button class="more-button" [matMenuTriggerFor]="menu" aria-label="Toggle menu">
-              <mat-icon>more_vert</mat-icon>
-            </button>
-            <mat-menu #menu="matMenu" xPosition="before">
-              <button mat-menu-item>Expand</button>
-              <button mat-menu-item>Remove</button>
-            </mat-menu>
-          </mat-card-title>
-        </mat-card-header>
-        <mat-card-content class="dashboard-card-content">
-          <div id='content'>
-          </div>
-        </mat-card-content>
-      </mat-card>
-    </mat-grid-tile>
-  </mat-grid-list>
+<div id="main-container">
+  <!-- Render plot. -->
+  <div id="plot-container">
+    <app-plot id="main-plot"></app-plot>
```

Comment:
is main-plot the name for the plot or is it defined somewhere what main-plot is?

Reply:
Just the name for the plot, I'll remove that since I stopped styling the <app-plot> directly in the CSS, instead I'm styling the <div id="plot-container"> that the plot lives in.